src/AppShed/Extensions/StorageBundle/Entity/Repository/DataRepository.php

```diff
@@ -21,7 +21,7 @@ class DataRepository extends EntityRepository
      */
     public function getDataForView(View $view)
     {
-        return $this->getFilteredData($view->getStore(), $view->getFilters());
+        return $this->getFilteredData($view->getStore(), new ArrayCollection($view->getFilters()->toArray()));
```

Comment:
no. I need to add extra items to collection, so i do 
  
    ArrayCollection(array_merge($api->getFilters()->toArray(), $additionalFilters))

I need ArrayCollection